vnext/codegen/react/components/rnwcore/ShadowNodes.h

```diff
@@ -94,7 +95,7 @@ using SwitchShadowNode = ConcreteViewShadowNode<
     SwitchProps,
 SwitchEventEmitter>;
 
-extern const char UnimplementedNativeViewComponentName[];
+JSI_EXPORT extern const char UnimplementedNativeViewComponentName[];
```

Comment:
>JSI_EXPORT [](http://example.com/codeflow?start=0&length=10)

I am a bit concerned that all these types are going to become as exported types from DLLs. These are not ABI-safe type. By adding this macro we effectively introduce a huge non-ABI safe API surface. Could avoid it? We will never be able to fix later.

Reply:
I dont think we export these:
https://github.com/microsoft/react-native-windows/blob/11602c2019871610bffb3e4dd5f2ec8969951809/vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj#L129